stock_inventory/views/stock_inventory.xml

```diff
@@ -150,5 +161,7 @@
         sequence="30"
         action="action_view_inventory_group_form"
     />
-    <delete model="ir.ui.menu" id="stock.menu_action_inventory_tree" />
+    <record id="stock.menu_action_inventory_tree" model="ir.ui.menu">
+        <field name="active" eval="False" />
+    </record>
```

Comment:
If we already have the module installed, the view does not exist anymore, so we can not access to "stock.menu_action_inventory_tree"

Reply:
Hi! You can just update the module "stock" and then install / update the "stock_inventory" module!

Reply:
Of course, but depending of the odoo hosting infrastructure, it is not a suitable / clean solution. Returning to the previous line of code or a small migration script will work without effort. what do you think ?

Reply:
"You can just update the module "stock" and then install / update the "stock_inventory" module!" - I confirm this works but it's not intuitive or documented anywhere except in this thread. A better solution needs to be found. Is it possible to force the upgrade of "stock" before upgrading this module?